docs/containerd-env-setup.md

```diff
@@ -4,23 +4,23 @@ This document will walk through how to setup a nydus image service to work with
 
 ## Install All Nydus Binaries
 
-Get `nydus-image`, `nydusd`, `nydusify`, `ctr-remote` and `nydus-overlayfs` binaries from [release](https://github.com/dragonflyoss/image-service/releases/latest) page.
+1. Get `nydus-image`, `nydusd`, `nydusify`, `ctr-remote` and `nydus-overlayfs` binaries from [release](https://github.com/dragonflyoss/image-service/releases/latest) page.
```

Comment:
ctr-remote should be deprecated? Nerdctl has already supported nydus. It would be better to remind user to install nydusctl

Reply:
We have one section about `ctr-remote` in the doc, should also remove it?

Reply:
I am suggesting removing it if no objections

Reply:
Removed.